scripts/deploy/main.json

```diff
@@ -380,7 +380,7 @@
             "value": "Warning"
           },
           {
-            "name": "Logging:LogLevel:SemanticKernel.Service",
+            "name": "Logging:LogLevel:CopilotChat",
```

Comment:
CopilotChat or ChatCopilot?

Reply:
CopilotChat, just for now until the rename is done (see below comment)